### PR TITLE
feat(legacy-plugin-chart-big-number): add control panel config for the BigNumber charts

### PR DESCRIPTION
**Pull request by suddjian**
💔 Breaking Changes

🏆 Enhancements

Allows BigNumber to register its own controls, so that the control configs can be removed from superset.

📜 Documentation

🐛 Bug Fix

🏠 Internal

### CODE REVIEW

File: plugins/big-number/src/BigNumber/index.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@
  */
 import { t } from '@superset-ui/translation';
 import { ChartMetadata, ChartPlugin } from '@superset-ui/chart';
+import controlPanel from './controls';
```
Comment (kristw) [inline plugins/big-number/src/BigNumber/index.ts:21]: Let's name the file `controlPanel` so it is the same when imported.

File: plugins/legacy-preset-chart-nvd3/src/NVD3Controls.tsx [hunk 2]
```diff
@@ -19,9 +19,12 @@
...
 import { t } from '@superset-ui/translation';
-import { controls } from '@superset-ui/chart';
-
-const { formatSelectOptions, D3_TIME_FORMAT_OPTIONS, D3_FORMAT_DOCS, D3_FORMAT_OPTIONS } = controls;
+import {
+  formatSelectOptions,
+  D3_TIME_FORMAT_OPTIONS,
+  D3_FORMAT_DOCS,
+  D3_FORMAT_OPTIONS,
+} from '@superset-ui/chart';
```
Comment (kristw) [inline plugins/legacy-preset-chart-nvd3/src/NVD3Controls.tsx:27]: Please see #417. May have to change reference after that one is merged.

File: plugins/legacy-preset-chart-nvd3/src/NVD3Controls.tsx [hunk 3]
```diff
@@ -338,6 +341,7 @@ export const timeSeriesSection = [
         'of query results',
     ),
     controlSetRows: [
+      // eslint-disable-next-line react/jsx-key
```
Comment (ktmud) [inline plugins/legacy-preset-chart-nvd3/src/NVD3Controls.tsx:344]: Can move this to top of file to disable the rule for the whole file.

Comment (suddjian) [reply plugins/legacy-preset-chart-nvd3/src/NVD3Controls.tsx:344]: I'm wary of disabling rules for a whole file, it's safer to do it per-line.

File: plugins/legacy-preset-chart-nvd3/src/NVD3Controls.tsx (hunk 2 above)
Comment (suddjian) [reply plugins/legacy-preset-chart-nvd3/src/NVD3Controls.tsx:27]: Let's merge that and I'll rebase

File: plugins/legacy-preset-chart-nvd3/src/NVD3Controls.tsx (hunk 3 above)
Comment (ktmud) [reply plugins/legacy-preset-chart-nvd3/src/NVD3Controls.tsx:344]: For this particular case, I don't understand why the rule needs to be disabled though. I see no real code change and it certainly has passed previous CI check?

Comment (kristw) [reply plugins/legacy-preset-chart-nvd3/src/NVD3Controls.tsx:344]: previously it was in the `temporary-plugins` dir which is excluded from CI build and tests

**Comment by suddjian** [comment]
Addressed PR feedback

File: plugins/big-number/package.json [hunk 4]
```diff
@@ -37,6 +37,7 @@
   "peerDependencies": {
     "@superset-ui/chart": "^0.13.0",
     "@superset-ui/color": "^0.13.0",
+    "@superset-ui/control-utils": "^0.0.0",
```
Comment (kristw) [inline plugins/big-number/package.json:40]: This should be `^0.13.0` now.

**Pull request merged**